TrackParamFilter/sections/specific.txt

```diff
@@ -129,6 +129,9 @@
 ||cam4.com^$removeparam=showSignupPopup
 ! Tracks source site
 ||office.com^$removeparam=ocid
+! Embedded tweets
+||twitter.com^*?ref_src=$removeparam=ref_src
+||twitter.com^*&ref_url=$removeparam=ref_url
```

Comment:
Let's try these rules
```
||twitter.com^$removeparam=ref_src
||twitter.com^$removeparam=ref_url
```

Reply:
OK, done.

Reply:
Thanks, I don't notice any incorrect blocking for now, but if you'll find it, please let us know.